### PR TITLE
docs: fix the wrongly written in advanced.zh-CN.md

### PR DESCRIPTION
**Pull request by Turkyden**
```diff
- 发布的 npm 包自然是资产，但编写的 TypeScript 类型定义、精心准备的组件库 demo 也都是资产
+ 发布的 npm 包自然是资产，编写的 TypeScript 类型定义、精心准备的组件库 demo 也都是资产
```

好像这里没有 ~但~ 字更加句子通顺哦。

### 🤔 这个变动的性质是？/ What is the nature of this change?

- [ ] 新特性提交 / New feature
- [ ] bug 修复 / Fix bug
- [ ] 样式优化 / Style optimization
- [ ] 包体积优化 / Package size optimization
- [ ] 性能优化 / Performance optimization
- [ ] 代码风格优化 / Code style optimization
- [ ] 构建优化 / Build optimization
- [x] 网站、文档、Demo 改进 / Website, documentation, demo improvements
- [ ] 测试相关 / Test related
- [ ] 其他 / Other

### 📝 更新日志 / Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     docs: fix the wrongly written in advanced.md     |
| 🇨🇳 Chinese |      docs: fix the wrongly written in advanced.zh-CN.md      |

### CODE REVIEW

File: docs/guide/advanced.zh-CN.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ toc: menu
 
 如何理解资产？从开发者视角狭义的理解，只要是生产出来可以帮助下游提效的实体，都可以称之为资产，比如组件、文档、组件 API、组件 demo 等等。
 
-我们在组件研发的过程中，无时无刻不在创建着资产。发布的 npm 包自然是资产，但编写的 TypeScript 类型定义、精心准备的组件库 demo 也都是资产，现在只需一行命令，即可将 dumi 与开发者共同完成的资产数据化，这份数据可以跟随 npm 包迭代、发布，进而流转给下游工具使用。
+我们在组件研发的过程中，无时无刻不在创建着资产。发布的 npm 包自然是资产，编写的 TypeScript 类型定义、精心准备的组件库 demo 也都是资产，现在只需一行命令，即可将 dumi 与开发者共同完成的资产数据化，这份数据可以跟随 npm 包迭代、发布，进而流转给下游工具使用。
```
Comment (xrkffgg) [inline docs/guide/advanced.zh-CN.md:69]: 英文 也瞅一下

Comment (Turkyden) [reply docs/guide/advanced.zh-CN.md:69]:
```diff
+ The released npm package is naturally an asset, and the written TypeScript type definitions and carefully prepared component library demos are also assets.
- The released npm package is naturally an asset, but the written TypeScript type definitions and carefully prepared component library demos are also assets.
```

@xrkffgg 2732eab

**Review by PeachScript — APPROVED** [review]

File: docs/guide/advanced.zh-CN.md (hunk 1 above)
Comment (PeachScript) [reply docs/guide/advanced.zh-CN.md:69]: 大多是 @xrkffgg Google 翻译 + 手动优化的

Comment (PeachScript) [reply docs/guide/advanced.zh-CN.md:69]: 你一看就是文法老司机，有时间的话 review 一下呗[抱大腿]

Comment (Turkyden) [reply docs/guide/advanced.zh-CN.md:69]: OK, haha! @PeachScript

**Pull request merged**